packages/charts/react-charting/src/utilities/utilities.ts

```diff
@@ -13,7 +13,10 @@ import {
   scaleBand as d3ScaleBand,
   scaleUtc as d3ScaleUtc,
   scaleTime as d3ScaleTime,
+  scaleLog as d3ScaleLog,
   NumberValue,
+  ScaleContinuousNumeric,
+  ScaleTime,
```

Comment:
we should rename this import. Hard to distinguish from d3ScaleTime

Reply:
done